source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,130 @@
 See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release. See the [Legacy Self-Hosted Mattermost Changelog](legacy-self-hosted-changelog) for details on all Mattermost self-hosted releases prior to v5.37. 
 
 Latest Mattermost Releases:
+- [Release v7.0 - Feature Release](#release-v7-0-feature-release)
 - [Release v6.7 - Feature Release](#release-v6-7-feature-release)
 - [Release v6.6 - Feature Release](#release-v6-6-feature-release)
 - [Release v6.5 - Feature Release](#release-v6-5-feature-release)
-- [Release v6.4 - Feature Release](#release-v6-4-feature-release)
 - [Release v6.3 - Extended Support Release](#release-v6-3-extended-support-release)
 
+## Release v7.0 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
+
+**Release Day: 2022-06-15**
+
+### Important Upgrade Notes
+ - MySQL self-hosted customers may notice the migration taking longer than usual when having a large number of rows in the ``FileInfo`` table.
```

Comment:
@harshilsharma63 Has this been tested on the community servers (if needed)?

Reply:
@agnivade what a DB spec? I've never done any of this before.

Reply:
@amyblais - Please add that.

```
The above was performed on a machine of specifications:
CPU - Intel i7 6 cores @ 2.6 GHz
Memory - 16 GB
```